LabeledIcon/LabeledIcon.js

```diff
@@ -29,9 +29,14 @@ const LabeledIconBase = kind({
 		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired
 	},
 
+	styles: {
+		css: componentCss,
+		className: 'labeledIcon',
+		publicClassNames: true
+	},
+
 	render: (props) => UiLabeledIcon.inline({
 		...props,
-		css: componentCss,
```

Comment:
To send the `css` prop down to `UiLabeledIcon` we must (always) manually extract it from the props and add it back. It's not included in the spread operation:
```
render: ({css, ...rest}) => UiLabeledIcon.inline({
	...rest,
	css,
```

Reply:
Alternatively, you can skip deconstructing and reference it directly.
```
render: (props) => UiLabeledIcon.inline({
  ...props,
  css: props.css
});
```